dom/attr/attr.js

```diff
@@ -84,22 +84,28 @@ var formElements = {"INPUT": true, "TEXTAREA": true, "SELECT": true},
 			}
 		}
 	},
-	setChildOptions = function(el, value){
-		if(value != null) {
-			var child = el.firstChild,
-				hasSelected = false;
-			while(child) {
-				if(child.nodeName === "OPTION") {
-					if(value === child.value) {
-						hasSelected = child.selected = true;
-						break;
-					}
-				}
-				child = child.nextSibling;
+	_findOptionToSelect = function (parent, value) {
+		var child = parent.firstChild;
+		while (child) {
+			if (child.nodeName === 'OPTION' && value === child.value) {
```

Comment:
our stuff runs in IE9, so this is ok: https://travis-ci.org/canjs/can-util/builds/222095765#L1154